backend/apps/matching/templates/map/map.html

```diff
@@ -26,8 +26,8 @@
     bURL     : "{% url 'participant_JSON' 'B' %}",
     aListURL  : "{% url 'participant_list' 'A' 'COUNTRYCODE' 'PLZ' 0 %}",
     bListURL   : "{% url 'participant_list' 'B' 'COUNTRYCODE' 'PLZ' 0 %}",
-    tileserver: "{{ tileserver }}",
-    mapboxToken       : "{{ mapbox_token }}",
+    tileURL: "{{ tileserver }}",
+    mapAttribution : `{{ attribution }}`,
```

Comment:
https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#safe
Need to add `| safe` filter

Reply:
thanks :)